src/main/java/com/blackducksoftware/integration/hub/alert/audit/repository/relation/AuditNotificationRelationPK.java

```diff
@@ -27,9 +27,25 @@
 
 @SuppressWarnings("serial")
 public class AuditNotificationRelationPK implements Serializable {
-    public Long auditEntryId;
-    public Long notificationId;
+    private Long auditEntryId;
```

Comment:
Did you test this? There seemed to be some weird behavior when these were private.

Reply:
It actually could have been because there was a constructor with the fields, but no setters. JPA didn't seem to like that.

Reply:
it works correctly with the changes I have made with the set and get methods